vowpalwabbit/cbify.cc

```diff
@@ -97,6 +119,43 @@ float loss_csldf(cbify& data, std::vector<v_array<COST_SENSITIVE::wclass>>& cs_c
   return data.loss0 + (data.loss1 - data.loss0) * cost;
 }
 
+template <class T>
```

Comment:
Why is this needed? Should already be available in the memory header

Reply:
Removed.